City-of-Helsinki/drupal-hdbt
# UHF-5573: News item image adjustment on teaser

# Adjusting news item image size on teaser [UHF-5573](https://helsinkisolutionoffice.atlassian.net/browse/UHF-5573)
Change the news item image on teaser from 30% wide to 25% wide.

## What was done
* Changed the news item image on teaser from 30% wide to 25% wide.
* Fixed the news-teaser-generic template to work with class arrays as well.
* Added missing class for news item teaser content.

## How to install
* Make sure your etusivu instance is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh`
* Update the HDBT theme
    * `composer require drupal/hdbt:dev-UHF-5573_news_item_image_adjustment`
* Run `make drush-cr`

## How to test
* Go to news archive on your local https://helfi-etusivu.docker.so/fi/news
* Make sure the layout doesn't break on mobile and that the image is 25% wide on the teaser.
